### PR TITLE
Plans: fall back to domain upgrade nudge if domain tip cannot be shown

### PR DESCRIPTION
**Pull request by gwwar**
As requested by @mtias this PR may fall back to a domain upgrade nudge if a domain tip cannot be shown.

Fallback:
domain-upgrade
Domain Tip:
domain-tip

## Testing Instructions
- Switch to a site with a free plan
- the a/b test `domainsWithPlansOnly` is set to `original`
- Navigate to My Sites, you should see the fallback
- update `domainsWithPlansOnly` to `plansOnly`
- You should see a domain tip

cc @matias @rralian @retrofox

### CODE REVIEW

File: client/my-sites/domain-tip/index.jsx [hunk 1]
```diff
@@ -46,10 +46,21 @@ const DomainTip = React.createClass( {
 		return true;
 	},
 
+	domainUpgradeNudge() {
+		return (
+			<UpgradeNudge
+					title={ this.translate( 'Get a free Custom Domain' ) }
```
Comment (mtias) [inline client/my-sites/domain-tip/index.jsx]: less indenting?

File: client/my-sites/domain-tip/index.jsx [hunk 2]
```diff
@@ -46,10 +46,21 @@ const DomainTip = React.createClass( {
...
+					message={ this.translate( 'Custom domains are free when you upgrade to a Premium or Business plan.' ) }
+					feature={ FEATURE_CUSTOM_DOMAIN }
+					event="stats_insights_domain"
```
Comment (mtias) [inline client/my-sites/domain-tip/index.jsx]: Not sure if we want to hardcode this? Should we pass an event prop to domain tip? How would we differentiate different domain tips in different places?

**Comment by mtias** [comment]
Thanks for looking at this @gwwar.

File: client/my-sites/domain-tip/index.jsx (hunk 2 above)
Comment (mtias) [inline client/my-sites/domain-tip/index.jsx]: Perhaps the domain tip event is `domain_tip_`${ this.props.event }``

Comment (gwwar) [inline client/my-sites/domain-tip/index.jsx]: 👍 Good catch. I've added an event prop in 9e0b00d. Are we ok with slightly redundant event names? The tip itself will then have the event of `domain_tip_stats_insights_domain`

**Comment by rralian** [comment]
Tested and this looks and works well for me. Code looks good.

**Pull request merged**